libs/dal/src/repositories/message/message.schema.ts

```diff
@@ -112,6 +112,11 @@ const messageSchema = new Schema<MessageDBModel>(
       },
       data: Schema.Types.Mixed,
     },
+    _actorId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Subscriber',
+      index: true,
```

Comment:
This index could be removed no need to have it.

Reply:
removed